lib/slack_bot_server/server.rb

```diff
@@ -59,7 +59,7 @@ class SlackBotServer::Server
   def initialize(queue: SlackBotServer::LocalQueue.new)
     @queue = queue
     @bots = {}
-    @add_proc = -> (token) { SlackBotServer::SimpleBot.new(token: token) }
+    @add_proc = ->(token) { SlackBotServer::SimpleBot.new(token: token) }
```

Comment:
I'd prefer it if these formatting changes were in a separate pull request if you think they're important.

Reply:
This is fixed now, please check again.